silk/datadog_checks/silk/events.py

```diff
@@ -0,0 +1,25 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+ALERT_TYPES = {"INFO": "info", "ERROR": "error", "WARNING": "warning", "CRITICAL": "error"}
+
+
+class SilkEvent(object):
+    def __init__(self, raw_event, tags):
+        self.raw_event = raw_event
+
+        self.payload = {
+            "timestamp": self.raw_event['timestamp'],
+            "event_type": self.raw_event["labels"],
+            "source_type_name": 'silk',
+            "alert_type": ALERT_TYPES[self.raw_event["level"]],
+            "tags": tags[:],
+        }
```

Comment:
perhaps wrap this in a try/except? we should safe guard against index errors. Maybe this is a valid case to return a None payload

Reply:
I'll wrap it in an if/else, since if I change to `.get()` instead of `[<key>]`, `self.payload` would never be `None`.